### PR TITLE
Show queue results

### PR DESCRIPTION
**Pull request by Sergiayn**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

### CODE REVIEW

File: resources/views/queues/view.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            <tr>
+                <th>Queue</th>
+                <th>Message</th>
+            </tr>
+            </thead>
+            <tbody></tbody>
+        </table>
+    </div>
+    <div>
+        <strong class="titlePushes">Pusched <span>0</span> jobs</strong>
```
Comment (samdark) [inline resources/views/queues/view.html:16]:
```suggestion
        <strong class="titlePushes">Pushed <span>0</span> jobs</strong>
```

File: resources/views/queues/summary.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<div class="yii-debug-toolbar__block">
+    <a href="#" title="" data-key="queue.link">Queues
+        <span class="yii-debug-toolbar__label" data-key="queue.messages">0</span>
+        <span class="yii-debug-toolbar__label" data-key="queue.pushes">0</span>
+        <span class="yii-debug-toolbar__label" data-key="queue.statuses">0</span>
```
Comment (samdark) [inline resources/views/queues/summary.html:5]: Could use `title` here to display hints about what these numers mean. Also coloring it differently would be cool.

File: resources/views/queues/summary.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        const cStatuses = queue.countStatuses
+
+        if (0 === cMessages && 0 === cPushes && 0 === cStatuses)
+            link.parentElement.style.display = "none"
+
+        document.querySelector('[data-key="queue.messages"]').textContent = cMessages
+        document.querySelector('[data-key="queue.pushes"]').textContent = cPushes
+        document.querySelector('[data-key="queue.statuses"]').textContent = cStatuses
+
+        link.title = 'Count: processing messages: ' + cMessages + ', pushes: ' + cPushes + ', statuses: ' + cStatuses
```
Comment (samdark) [inline resources/views/queues/summary.html:24]: Separate titles for each number will likely be more helpful.

**Review by rustamwin — CHANGES_REQUESTED** [review]

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**